openfecwebapp/templates/macros/filters/range.html

```diff
@@ -0,0 +1,16 @@
+{% macro amount(name, label) %}
+<fieldset class="filter" id="{{name}}-field">
+  <legend class="label">{{ label }}</legend>
+  <div class="range">
+    <div class="range__input js-filter">
+      <label>From</label>
```

Comment:
Note that you can add a `for` attribute in a `label` element and set it to match the `id` of the input element.  This helps with [accessibility](https://pages.18f.gov/accessibility/forms/) as it ties the label to the form field. :-)